packages/types/src/pricing/common/price-rule.ts

```diff
@@ -48,6 +48,18 @@ export interface PriceRuleDTO {
    * The ID of the associated price list.
    */
   price_list_id: string
+  /**
+   * When the price_rule was created.
+   */
+  created_at?: null | Date
+  /**
+   * When the price_rule was updated.
+   */
+  updated_at?: null | Date
```

Comment:
question: those two are always present and with a default value no? This DTO is that one that reflect what is returned by the module by default right?